cmd/acikubectl/cmd/debug.go

```diff
@@ -368,7 +372,21 @@ func clusterReport(cmd *cobra.Command, args []string) {
 				args: nodeItem.argFunc(systemNamespace, podName,
 					nodeItem.cont, nodeItem.args),
 			})
+
+			//Prepare kubectl cp command for aci-conatiners-host
+			tempName = fmt.Sprintf("hostfiles/node-%s/aci-containers-host/k8s-pod-network/", node.Name)
```

Comment:
nit: this is still not the same as the original file system: aci-containers-host/k8s-pod-network versus the original file system which is: aci-containers/k8s-pod-network

Reply:
We still need this to be addressed.